elfo-core/src/atomic_status_kind.rs

```diff
@@ -0,0 +1,29 @@
+use std::mem;
+use std::sync::atomic::{self, AtomicU8};
+
```

Comment:
I think it's better to move `ActorStatusKind` (and maybe `ActorStatus`) here and name this file "actor_status.rs".

`permissions.rs`, for instance, contain both `Permissions` and `AtomicPermissions`.

Reply:
Done in [1070425](https://github.com/elfo-rs/elfo/pull/133/commits/1070425102a5c11c57e2f0617259ec6dbfb6c18c).